src/index.html

```diff
@@ -8,10 +8,172 @@
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            class="enter"
+            min="1"
+            max="100"
+            value="12"
+            name="age">
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            class="enter"
+            name="birthDate">
+        </label>
+
+        <label class="form-field lastInput">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            class="enter"
+            required>
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend> Registration </legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            class="enter"
+            name="mail"
+            placeholder="email@example.com"
+            required>
+        </label>
+
+        <label class="form-field lastInput">
+          Password:
+          <input
+            type="password"
+            class="enter"
+            name="password"
+            minlength="5"
+            maxlength="10"
+            required>
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend> An interesting fact about you! </legend>
+
+        <label class="form-field">
+          Do you love cats?
+          <input
+            type="radio"
+            class="enter"
+            name="cat"
+            value="y"> Yes
+          <input
+            type="radio"
+            class="enter"
+            name="cat"
+            value="n"> No
+        </label>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            class="enter"
+            name="color">
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            class="enter"
+            name="time"
+            step="2">
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select
+            name="cars"
+            id="cars"
+            multiple>
+
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field lastInput">
+          How do you rate our work?
+          <input
+            type="range"
+            class="enter"
+            name="range"
+            value="0">
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend> Additional info: </legend>
+        <label class="form-field">
+          Comments:
+          <textarea
+            name="comment"
+            cols="20"
+            rows="2">
+          </textarea>
```

Comment:
that's where I see my caret when I click on your text area. It happens because everything between opening and closing tags is considered as value for the textarea, which, in your case, is the blank line& Proper fix will be something like this
```suggestion
          <textarea
            name="comment"
            cols="20"
            rows="2"
          ></textarea>
```